src/main/java/org/jenkinsci/plugins/docker/commons/fingerprint/DockerFromFingerprintFacet.java

```diff
@@ -3,13 +3,30 @@
 import hudson.model.Fingerprint;
 
 /**
- * As a facet on a fingerprint that represents a docker image,
- * indicates that another image was produced by deriving from it. See "FROM" in Dockerfile.
- *  
- * @author Kohsuke Kawaguchi
+ * Facet representing the fact that one Docker image was derived from another.
+ * This corresponds to the {@code FROM} directive in a {@code Dockerfile}.
+ * As a facet on a fingerprint corresponding to the base image, refers to the derived image.
```

Comment:
terminology here is wrong.  A base image is the root of all the parents parent.  I think you mean the parent image?